backend/ops_api/ops/schemas/budget_line_items.py

```diff
@@ -270,6 +298,7 @@ class BudgetLineItemCANSchema(Schema):
 
 
 class SimpleAgreementSchema(Schema):
+    id = fields.Integer(required=True)
```

Comment:
this is just a reshuffle and not needing an update to the openApi spec?

Reply:
There are a number of updates needed to the openApi spec from this PR and the last PR that will need to get updated.  I think we will want to fix PROD first though.